packages/emitter-framework/src/csharp/components/property/property.tsx

```diff
@@ -20,10 +25,28 @@ export function Property(props: PropertyProps): Children {
   const result = preprocessPropertyType(props.type);
   const { $ } = useTsp();
 
+  let overrideType: "" | "override" | "new" = "";
+  if (props.type.model && props.type.model.baseModel) {
+    const base = props.type.model.baseModel;
+    for (const baseProperty of walkPropertiesInherited(base)) {
```

Comment:
Feels like could have a better function that doesn't list every single property here. I think just `getProperty` will lookup as well